build.gradle.kts

```diff
@@ -413,6 +413,8 @@ subprojects {
           artifact(javadocJar)
         }
 
+        artifactId = "${rootProject.name.lowercase(Locale.getDefault())}-${project.name}"
```

Comment:
Why the invocation to `Locale.getDefault()`?

Reply:
I was following the code style in this script, where `toLowerCase(Locale.getDefault())` is actually just `toLowerCase()`.

Reply:
Specifying `Local` for `lowercase ()` API is a best practice in JVM language, otherwise we will get an expected result.